pkg/kubectl/sorter.go

```diff
@@ -277,13 +277,16 @@ func (r *RuntimeSort) Less(i, j int) bool {
 	iObj := r.objs[i]
 	jObj := r.objs[j]
 
-	parser := jsonpath.New("sorting").AllowMissingKeys(true)
-	parser.Parse(r.field)
-
 	var iValues [][]reflect.Value
 	var jValues [][]reflect.Value
 	var err error
 
+	parser := jsonpath.New("sorting").AllowMissingKeys(true)
+	err = parser.Parse(r.field)
+	if err != nil {
+		glog.Fatalf("Failed to parse field:%s err: %v", r.field, err)
```

Comment:
In general, I prefer a panic.

If you try to allow missing keys, why wouldn't you allow a parsing failure?

Reply:
Done